.gitmodules

```diff
@@ -197,6 +197,9 @@
 [submodule "vendor/grammars/abap.tmbundle"]
 	path = vendor/grammars/abap.tmbundle
 	url = https://github.com/pvl/abap.tmbundle
+[submodule "vendor/grammars/vscode-abap-cds"]
+	path = vendor/grammars/vscode-abap-cds
+	url = https://github.com/SAP-samples/vscode-abap-cds
```

Comment:
> I made the PR by looking at other pull requests without actually getting the project to run. Hopefully everything is in order.

The new grammar should be added by using `scripts/add-grammar`.

Reply:
I don't have a linux machine at hand to get it running, but since the build is failing, I'll have to do it somehow.